components/profile/index.js

```diff
@@ -0,0 +1,6 @@
+import  { ViewProfile, EditProfile } from "./components";
+
+export {
+    ViewProfile,
+    EditProfile
+};
```

Comment:
rewrite this as:
```
import  * as components from "./components";

export { components };
```
Reasoning being you do not have to update this file every time you add a component. Secondly you will be exporting other objects such as `actions`, `sagas`, `selectors`, `reducers` from this file later on. Make sure to adhere to the pattern that is set out for you in the coding conventions. It is important that you conform to it because it will make you life a lot easier as your application grows.